website/source/docs/secrets/pki/index.html.md

```diff
@@ -1235,6 +1235,12 @@ subpath for interactive help output.
         This sets the O (Organization) values in the subject field of issued
         certificates. This is a comma-separated string.
       </li>
+      <li>
+        <span class="param">generate_lease</span>
+        <span class="param-flags">optional</span>
+        If set, certificates issued/signed against this role will have Vault
+        leases attached to them. Defaults to "false".
```

Comment:
Should probably discuss here how if leases are not attached, the pki/revoke endpoint will have to be used if a certificate should be added to the CRL rather than the normal Vault revocation paths. Maybe also something about if generating large numbers (tens of thousands+) of certs it's usually a good idea not to set this as it can impact Vault startup time.